tests/api/entities/resolver/resolve.test.js

```diff
@@ -51,6 +51,56 @@ describe('entities:resolve', () => {
     entries[0].edition.uri.should.equal(`isbn:${isbn}`)
   })
 
+  it('should resolve an edition with no ISBN to an existing edition, if the title matches', async () => {
+    const { isbn, claims } = await createEditionWithIsbn()
+    const editionTitle = claims['wdt:P1476'][0]
```

Comment:
maybe adding a case insensitive feature

```suggestion
    const editionTitle = claims['wdt:P1476'][0].toUpperCase()
```

Reply:
yes! done in 130cd7f9